package.json

```diff
@@ -4,7 +4,7 @@
   "description": "GraphicaL User Interface for creating and running Scratch 3.0 projects",
   "main": "./src/index.js",
   "scripts": {
-    "build": "npm run clean && webpack --progress --colors --bail",
```

Comment:
`npm run clean` runs `rm` which doesn't work on Windows. By removing the `npm run clean` step here, Windows users can at least `npm run build` even if `npm run clean` still won't work for them.